server/routes/api/documents.ts

```diff
@@ -185,57 +171,6 @@ router.post("documents.list", auth(), pagination(), async (ctx) => {
   };
 });
 
-router.post("documents.pinned", auth(), pagination(), async (ctx) => {
```

Comment:
No backwards compatibility here, the likelihood of anyone else using these particular API's is slim